### PR TITLE
chore: Remove Helm v2 support

### LINKED ISSUE #50: Question: Removal of Helm v2 support?
Helm [v2.17.0](https://github.com/helm/helm/releases/tag/v2.17.0) was the final release of Helm v2 in October 2020. Helm v2 is unsupported since November 2020, as detailed in [Helm 2 and the Charts Project Are Now Unsupported](https://helm.sh/blog/helm-2-becomes-unsupported/).

Helm v2 uses an older version of Kubernetes Go client which is incompatible with Kubernetes Go Client 1.18+. [helm-mapkubeapis](https://github.com/helm/helm-mapkubeapis) calls Helm v2 storage APIs which in turn call Kubernetes Go client APIs. This means that the plugin is restricted to using Helm [v3.1.3](https://github.com/helm/helm/releases/tag/v3.1.3) (last Helm v3 release which uses Kubernetes Go Client pre 1.18) which was released in April 2020.

Is it time to drop support for Helm v2 in this plugin as Helm v2 is no longer supported? The benefit is that we can update to the latest dependencies.

### PR DESCRIPTION
**Pull request by hickeyma**
Helm [v2.17.0](https://github.com/helm/helm/releases/tag/v2.17.0) was the final release of Helm v2 in October 2020. Helm v2 is unsupported since November 2020, as detailed in [Helm 2 and the Charts Project Are Now Unsupported](https://helm.sh/blog/helm-2-becomes-unsupported/).

Helm v2 uses an older version of Kubernetes Go client which is incompatible with Kubernetes Go Client 1.18+. [helm-mapkubeapis](https://github.com/helm/helm-mapkubeapis) calls Helm v2 storage APIs which in turn call Kubernetes Go client APIs. This means that the plugin is restricted to using Helm [v3.1.3](https://github.com/helm/helm/releases/tag/v3.1.3) (last Helm v3 release which uses Kubernetes Go Client pre 1.18) which was released in April 2020.

It therefore makes sense to remove Helm v2 support.

Closes #50 

Signed-off-by: Martin Hickey <martin.hickey@ie.ibm.com>

### CODE REVIEW

File: cmd/mapkubeapis/map.go [hunk 1]
```diff
@@ -36,7 +35,6 @@ type MapOptions struct {
 	MapFile          string
 	ReleaseName      string
 	ReleaseNamespace string
-	RunV2            bool
 	StorageType      string
 	TillerOutCluster bool
```
Comment (marckhouzam) [inline cmd/mapkubeapis/map.go:39]: Do you want to also remove the `StorageType` and `TillerOutCluster` fields here?

If so, they can probably also be removed in `common.go`.

**Review by marckhouzam — APPROVED** [review]
This looks right to me.  I haven't tested it on a cluster however, but I'm not worried.

Will you be updating the plugin version in a separate PR?

File: cmd/mapkubeapis/map.go (hunk 1 above)
Comment (hickeyma) [reply cmd/mapkubeapis/map.go]: Sure, good catch. Removed.

**Comment by hickeyma** [comment]
Thanks for the review @marckhouzam. I have updated with your comments.

> Will you be updating the plugin version in a separate PR?

Lets do it in this PR. Bumped to "0.3.0"/

**Review by marckhouzam — APPROVED** [review]
LGTM

**Review by mattfarina — APPROVED** [review]
I ran some quick tests and "I think" it's working ok.

**Pull request merged**